editor/modes/visual-editor/invalid-block-warning.js

```diff
@@ -54,6 +62,12 @@ export default connect(
 	null,
 	( dispatch, ownProps ) => {
 		return {
+			ignoreInvalid() {
+				const { block } = ownProps;
+				const serializedBlock = serialize( { ...block, isValid: true } );
+				const parsedBlocks = parse( serializedBlock );
+				dispatch( replaceBlock( block.uid, parsedBlocks ) );
```

Comment:
>Why can't we just set `isValid` to true? where it's being overridden?

Yeah, I had considered it. Certainly the most direct approach. It might have been the direct mutation that concerned me. Let me see how it works out.